### ISSUE TITLE
Allow to toggle cache feature on / off

### ISSUE THREAD

**Issue by EranIsraeli** [springfox/springfox#863]

**Comment by dilipkrish** [comment]
@EranIsraeli Its been removed altogether because of problems details here -> #858 

When I say removed, I mean it no longer interferes with consumer caching choices. It maintains an internal cache. Would love feedback based on 2.1.2-SNAPSHOT thats been pushed

**Comment by EranIsraeli** [comment]
@dilipkrish I've experienced some regression going from version 2.0.0 to version 2.1.1. This is only one of about 4 other issues I've ran into.

Regarding the cache issue -  I'm using ehcache and although I've added the required caches that springfox uses to my ehcache.xml (models, modelDependencies, modelProperties, operations), springfox is still trying to create a new cache manager..... 

I'll check out the 2.1.2-SNAPSHOT. 

Thanks.
